### PR TITLE
increase timeout wpt

### PR DESCRIPTION
**Pull request by FeliceGeracitano**
a test can take up to `2 min` now that we have `runs = 3` as default, increase timeout for wpt test

### CODE REVIEW

File: packages/inspector-wpt/src/const.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const DEFAULT_AUDIT = {
   fvonly: 0
 };
 export const APP_PORT = process.env.APP_PORT || 3000;
-export const WPT_TIMEOUT = 60000 * 5;
+export const WPT_TIMEOUT = 60000 * 30; // 30 mimutes
```
Comment (rbelling) [inline packages/inspector-wpt/src/const.ts]: should this be 29 minutes since the cron runs every 30 minutes?

Comment (FeliceGeracitano) [reply packages/inspector-wpt/src/const.ts]: i think we set to run every hour now:
Screenshot 2019-03-19 at 11 20 47

Comment (rbelling) [reply packages/inspector-wpt/src/const.ts]: oh sorry forgot that. Ok!

Comment (FeliceGeracitano) [reply packages/inspector-wpt/src/const.ts]: This is only our node server timeout to poll wpt result and write points into DB, wpt-server & wpt-agent timeout are managed differently and we should investigate in their documentation

**Pull request merged**